lib/linguist/generated.rb

```diff
@@ -537,7 +537,9 @@ def generated_grpc_cpp?
     def generated_dart?
       return false unless extname == '.dart'
       return false unless lines.count > 1
-      return lines[0].include?("GENERATED CODE - DO NOT MODIFY")
+      genStatement = lines[0].include?("Generated code. Do not modify.")
+      getStatementCaps = lines[1].include?("GENERATED CODE - DO NOT MODIFY")
```

Comment:
What about this?
```ruby
return true if lines[0].include?("Generated code. Do not modify.")
return lines[1].include?("GENERATED CODE - DO NOT MODIFY")
```
If we're going to run this on all `.dart` files of more than 1 line, we might as well make it as fast as we can.